modular_zubbers/code/modules/storyteller/storytellers/tellers/storyteller_2_default.dm

```diff
@@ -1,10 +1,11 @@
 /datum/storyteller/default
 	name = "Default Andy (Medium Chaos)"
 	desc = "Default Andy is the default Storyteller, and the comparison point for every other Storyteller. \
-	More frequent events than the Chill or the Fragile, but less frequent events than The Gamer or the Clown. Best for an average, varied experience."
+	More frequent events than Chill, but less frequent events than Gamer. Best for an average, varied experience."
 	welcome_text = "If I chopped you up in a meat grinder..."
```

Comment:
Probably worth removing welcome text since we don't use it anymore

Reply:
Going to keep it as part of configurable secret/not secret.